models/turbine_models/custom_models/sd_inference/utils.py

```diff
@@ -331,7 +340,8 @@ def get_wmma_spec_path(target_chip, save_dir, masked_attention=False):
     else:
         return None
     attn_spec = urlopen(url).read().decode("utf-8")
```

Comment:
I suppose this isn't the only place where we are fetching network resources, but wish we weren't doing this... some day.